### ISSUE TITLE
Update doesn't check back to cve.startyear

### ISSUE THREAD

**Issue by colezlaw** [dependency-check/DependencyCheck#272]
When setting `cve.startyear` via properties to something very recent, if you change that date back to the original 2002, the database check only ensures that the most _recent_ data exists, but does not check to ensure the oldest data exists.

**Comment by jeremylong** [comment]
This would be a rare failure case as the `cve.startyear` is not exposed as an argument in any of the interfaces (cli/maven/etc.).

**Comment by colezlaw** [comment]
Right - I only ran into it because when I was making the homebrew formula, they wanted a more substantial test than just dependency-check - v - but anything more substantial requires CVE data (maybe another thing I’d like to address as part of the exception refactoring - the evidence alone is valuable information, even if you don’t get CVE’s) - but downloading all the NIST data initially takes a very long time, so the package generates a properties file with start year set to 2015 then scans the libexec folder. This ended up being a problem INITIALLY because I didn’t set the data directory - I had wanted it to use the default. The problem was that since it used the default, the first time you actually USED dependency-check, DC thought it was up to date but only had 2015 and newer data.

I worked around it in the formula by just putting the data into the temp directory that hombrew creates for the test itself. Problem solved.
